async-api.yml

```diff
@@ -100,6 +102,8 @@ channels:
         $ref: '#/components/messages/prison-offender-events.prisoner.restriction.changed'
       prison-offender-events.prisoner.sentence-dates-changed:
         $ref: '#/components/messages/prison-offender-events.prisoner.sentence-dates-changed'
+      prison-offender-events.video-appointment.cancelled:
+        $ref: '#/components/messages/prison-offender-events.prisoner.video-appointment.cancelled'
```

Comment:
this is odd - I think you've got the message as `prison-offender-events.prisoner.video-appointment.cancelled` but you've also got it as `prison-offender-events.video-appointment.cancelled`.  Probably best to have the `prisoner` in rather than out

Reply:
Looks like a copy error. Will sort, thanks.

Reply:
Consistently named now.

Reply:
Using `prison-offender-events.prisoner.video-appointment.cancelled`